### ISSUE TITLE
Course preview pages need a title

### ISSUE THREAD

**Issue by fofr** [DFE-Digital/manage-courses-ui#160]
The title is currently:
`<title>Publish teacher training courses - GOV.UK</title>`

It should be:
`<title>Preview of course [course name]([course code]) - Publish teacher training courses - GOV.UK</title>`

eg
`Preview of Chemistry (ABC1)`

**Comment by fofr** [comment]
This also applies to the child pages when editing a course.
